kinto/core/initialization.py

```diff
@@ -8,7 +8,7 @@
 from pyramid.events import NewRequest, NewResponse
 from pyramid.exceptions import ConfigurationError
 from pyramid.httpexceptions import (HTTPTemporaryRedirect, HTTPGone,
-                                    HTTPBadRequest)
+                                    HTTPBadRequest, HTTPUnauthorized)
```

Comment:
?

Reply:
I accidentally let this slip in from a previous commit.  I will get rid of it!